modules/sw360/sw360-client/src/main/java/org/eclipse/sw360/antenna/sw360/rest/resource/releases/SW360Release.java

```diff
@@ -338,12 +340,26 @@ public SW360Release mergeWith(SW360Release releaseWithPrecedence) {
         if (componentIdWithPrecedence != null && !componentIdWithPrecedence.getHref().isEmpty()) {
             get_Links().setSelfComponent(componentIdWithPrecedence);
         }
+        final List<SW360SparseAttachment> releaseWithPrecedenceAttachments = releaseWithPrecedence.get_Embedded().getAttachments();
+        if (releaseWithPrecedenceAttachments != null && !releaseWithPrecedenceAttachments.isEmpty()) {
+            if (get_Embedded().getAttachments() == null || get_Embedded().getAttachments().isEmpty()) {
```

Comment:
SW360ReleaseEmbedded.getAttachments() never returns null, so the null checks can be removed.